modules/ROOT/Edits/partner-manager-delete-partner.adoc

```diff
@@ -0,0 +1,26 @@
+= Delete a Partner
+
+If you are no longer trading with a partner, you can delete the partner.
+
+When you delete a partner, the partner is removed from the Partner Manager's *Partners* page, and all the partner's data is deleted, except for activities for the partner that occurred before the partner was deleted.
+
+[NOTE]
+--
+If you are deleting a partner from an on-premises installation, the server must be running before you can delete a partner.
+
+Contact your B2B System Administrator to ensure that your on-premises Mule instances, the servers hosting them, and the server group in Anypoint Runtime Manager called `B2BserverGroup` are active and running.
+
+You can also check the status of the server in Anypoint Platform Runtime Manager.
+--
+
+== Steps to Delete a Partner
```

Comment:
GLOBALLY search through doc to review consistency amount topic titles identifying "Steps"; some are that term only.